src/uiHelpers.h

```diff
@@ -8,71 +8,126 @@
 typedef void ui_callback_fn_t();
 
 typedef enum {
-	CALLBACK_NOT_RUN,
-	CALLBACK_RUN,
+    CALLBACK_NOT_RUN,
+    CALLBACK_RUN,
 } ui_callback_state_t;
 
 typedef struct {
-	ui_callback_state_t state;
-	ui_callback_fn_t* confirm;
-	ui_callback_fn_t* reject;
+    ui_callback_state_t state;
+    ui_callback_fn_t* confirm;
+    ui_callback_fn_t* reject;
 } ui_callback_t;
 
 
 typedef struct {
-	uint16_t initMagic;
-	char header[30];
-	char currentText[18];
-	char fullText[200];
-	size_t scrollIndex;
-	ui_callback_t callback;
-	#ifdef HEADLESS
-	bool headlessShouldRespond;
-	#endif
+    uint16_t initMagic;
+    char header[30];
+    char currentText[18];
+    char fullText[200];
+    size_t scrollIndex;
+    ui_callback_t callback;
+#ifdef HEADLESS
+    bool headlessShouldRespond;
+#endif
 } scrollingState_t;
 
 typedef struct {
-	uint16_t initMagic;
-	char header[30];
-	char text[30];
-	ui_callback_t callback;
-	#ifdef HEADLESS
-	bool headlessShouldRespond;
-	#endif
+    uint16_t initMagic;
+    char header[30];
+    char text[30];
+    ui_callback_t callback;
+#ifdef HEADLESS
+    bool headlessShouldRespond;
+#endif
 } confirmState_t;
 
 typedef union {
-	scrollingState_t scrolling;
-	confirmState_t confirm;
+    scrollingState_t scrolling;
+    confirmState_t confirm;
 } displayState_t;
 
 
 void ui_idle(void);
 
 void ui_displayScrollingText(
-        const char* headerStr,
-        const char* bodyStr,
-        ui_callback_fn_t* callback);
+    const char* headerStr,
+    const char* bodyStr,
+    ui_callback_fn_t* callback);
 
 void ui_displayConfirm(
-        const char* headerStr,
-        const char* bodyStr,
-        ui_callback_fn_t* confirm,
-        ui_callback_fn_t* reject
+    const char* headerStr,
+    const char* bodyStr,
+    ui_callback_fn_t* confirm,
+    ui_callback_fn_t* reject
 );
 
 void ui_checkUserConsent(
-        security_policy_t policy,
-        const char* headerStr,
-        const char* bodyStr,
-        ui_callback_fn_t* confirm,
-        ui_callback_fn_t* reject
+    security_policy_t policy,
+    const char* headerStr,
+    const char* bodyStr,
+    ui_callback_fn_t* confirm,
+    ui_callback_fn_t* reject
 );
 
 void ui_displayBusy();
+void ui_displayConfirm_init();
```

Comment:
Can be refactored later but we might rename this to
`ui_displayPrompt` and `ui_displayText`